README.md

```diff
@@ -1,2 +1,13 @@
 # swc-conda-recipes
 Conda meta-packages for helping with workshop installation
+
+Core concepts:
+  * environment.yml should be downloaded by end-users into a folder by itself, followed by
+
+```
+conda-env create 
```

Comment:
Oddly enough, these seem to be equivalent.  I think conda env calls conda-env.  I think I'll change it to your suggestion, though, because that's more consistent with the way other conda add-ons work (and how they're documented).